src/flick/show/utils.py

```diff
@@ -33,46 +36,65 @@
 """
 
 
-def get_movie_from_DBinfo(info):
+def get_movie_from_DBinfo(info, credit):
     """
     get a flick movie object similar by parsing the
     information returned by movieDB
     """
     genres = info.get("genres")
-    tags = [genre.get("name") for genre in genres] if genres else []
+    crew = credit.get("crew")
+    cast = credit.get("cast")
+    tags = [g.get("name") for g in genres] if genres else []
+    castInfo = [p.get("name") for p in cast] if cast else []
+    directors = []
+    for c in crew:
+        if c.get("job") == "Director":
+            directors.append(c.get("name"))
 
     movie = {
         "show_id": info.get("id"),
         "title": info.get("original_title"),
-        "poster_pic": info.get("poster_path"),
+        "poster_pic": BASE_MOVIE_URL + info.get("poster_path"),
         "show_tags": tags,
         "is_tv": False,
         "date_released": info.get("release_date"),
         "duration": info.get("runtime"),
         "language": info.get("original_language"),
         "description": info.get("overview"),
+        "director(s)": directors,
```

Comment:
better to rename this as "directors" -- parenthesis is a bit risky because we end up using other languages it'll be weird

Reply:
ie strings could be translated into chinese